design/root.go

```diff
@@ -0,0 +1,39 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+	)
+
+var root = a.Type("Root", func() {
```

Comment:
Your root is missing a type. See the labels design for an example here: https://github.com/fabric8-services/fabric8-wit/blob/a4d9061b1b35a4e7acef28da0dc4bc80f39a72bb/design/labels.go#L10

```go
	a.Attribute("type", d.String, func() {
		a.Enum("rootendpoints")
	})
```